src/content/docs/new-relic-solutions/new-relic-one/introduction-new-relic-platform.mdx

```diff
@@ -0,0 +1,216 @@
+
+## Understanding connections [#connections]
+
+One of the most powerful aspects of New Relic is how it lets you tie together all the components in your system and understand how they're connected. Let's look at a few features related to that. 
+
+### Distributed tracing [#tracing]
+
+Distributed tracing is powerful for helping you dig down into a specific kind of transaction and see the various calls and components involved in that activity. For more on using this feature, see [Distributed tracing](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui). 
+
+There are a few ways to get to distributed tracing: 
+
+* Your global distributed tracing. Go to: **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Traces**. This shows you traces from across your accounts and entities. 
+* Entity-specific tracing. You can also go to a specific monitored entity (for example, an APM-monitored app) and click **Distributed tracing** there. That shows you traces that involve that specific entity. 
+
+<img
+  title="Distributed tracing trace waterfall"
+  alt="Distributed tracing trace waterfall"
+  src={traceWaterfall}
+/>
+### Service maps [#maps]
+
+When you're in the UI for a specific monitored entity, you can click **Service map** to see a visual map of how that service relates to other monitored entities. For more details, see [Service maps](/docs/new-relic-solutions/new-relic-one/ui-data/service-maps/introduction-service-maps).
+
+<img
+  title="APM service map"
+  alt="APM service map"
+  src={serviceMap}
+/>
+
+### Workloads [#workloads]
+
+Our [workloads](/docs/new-relic-solutions/new-relic-one/workloads/workloads-isolate-resolve-incidents-faster) feature lets you group together related entities in ways that make sense to your business. This becomes more and more important the bigger your organization is. Here's one of the UI pages you can find in a workload: a map showing the workload entities. 
+
+<img
+  title="Workload map"
+  alt="Workload map"
+  src={workloadMap}
+/>
+
+## Query your data [#query]
+
+<img
+  title="nrql_example.png"
+  alt="nrql_example.png"
+  src={queriesnrqlNrqlExampleTimeseries}
+/>
+
+Learning how to query your data is hugely important to getting the most out of New Relic. We give you a lot of out-of-the-box UI experiences, but to optimize your observability endeavors you'll need to bring in some custom data and make some custom dashboards. 
+
+You don't have to learn NRQL yet. Once you're feeling comfortable with New Relic and are ready to unlock more of its power, you can start delving into [our NRQL tutorials](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language).
```

Comment:
This seems like a sudden jump, maybe some kind of lead-in about NRQL or querying before this sentence?

Reply:
The link says NRQL tutorials but goes to the introduction. Maybe just NRQL docs instead of NRQL tutorials?